admin_guide/node_problem_detector.adoc

```diff
@@ -0,0 +1,426 @@
+                        "pattern": "Error trying v2 registry: failed to register layer: rename /var/lib/docker/image/(.+) /var/lib/docker/image/(.+): directory not empty.*"
+                }
+        ]
+    }
+  kernel-monitor.json: |
+    {
+        "plugin": "journald",
+        "pluginConfig": {
+                "source": "kernel"
+        },
+        "logPath": "/host/log/journal",
+        "lookback": "5m",
+        "bufferSize": 10,
+        "source": "kernel-monitor",
+        "conditions": [
+                {
+                        "type": "KernelDeadlock",
+                        "reason": "KernelHasNoDeadlock",
+                        "message": "kernel has no deadlock"
+                }
+        ],
+        "rules": [
+                {
+                        "type": "temporary",
+                        "reason": "OOMKilling",
+                        "pattern": "Kill process \\d+ (.+) score \\d+ or sacrifice child\\nKilled process \\d+ (.+) total-vm:\\d+kB, anon-rss:\\d+kB, file-rss:\\d+kB"
+                },
+                {
+                        "type": "temporary",
+                        "reason": "TaskHung",
+                        "pattern": "task \\S+:\\w+ blocked for more than \\w+ seconds\\."
+                },
+                {
+                        "type": "temporary",
+                        "reason": "UnregisterNetDevice",
+                        "pattern": "unregister_netdevice: waiting for \\w+ to become free. Usage count = \\d+"
+                },
+                {
+                        "type": "temporary",
+                        "reason": "KernelOops",
+                        "pattern": "BUG: unable to handle kernel NULL pointer dereference at .*"
+                },
+                {
+                        "type": "temporary",
+                        "reason": "KernelOops",
+                        "pattern": "divide error: 0000 \\[#\\d+\\] SMP"
+                },
+                {
+                        "type": "permanent",
+                        "condition": "KernelDeadlock",
+                        "reason": "AUFSUmountHung",
+                        "pattern": "task umount\\.aufs:\\w+ blocked for more than \\w+ seconds\\."
+                },
+                {
+                        "type": "permanent",
+                        "condition": "KernelDeadlock",
+                        "reason": "DockerHung",
+                        "pattern": "task docker:\\w+ blocked for more than \\w+ seconds\\."
+                }
+        ]
+    }
+----
+
+. Remove or add any NodeConditions (conditions) or events (rules) as needed. 
+
+. Restart running pods to apply the changes. To restart pods, you can delete all existing pods:
++
+[source,bash]
+----
+# oc delete pods -n openshift-node-problem-detector -l name=node-problem-detector
+----
+
+[[admin-guide-node-problem-detector-run]]
+== Reinstalling the Node Problem Detector
```

Comment:
This section is called "Reinstalling", but I don't think we ever tell how to install it (excepting the manual install section below). The "reinstall" instructions work for an "install" too, or we can tell them to set the inventory variable `openshift_node_problem_detector_install=true` prior to running the OpenShift installer. Maybe that belongs in this section https://docs.openshift.com/container-platform/3.9/install_config/install/advanced_install.html#configuring-ansible or elsewhere. I'm not familiar with our docs conventions.